analyses/ewings-cell-type/references/README.md

```diff
@@ -0,0 +1,16 @@
+# Reference files
+
+This directory contains reference files used for cell typing Ewing sarcoma samples from SCPCP000015.
+
+## Tumor marker genes
+
+`tumor_marker_genes.tsv` contains a list of marker genes used to identify tumor cells in previously published studies.
+Currently, this list only contains genes that should mark all tumor cells and we are not yet considering subsets of tumor cells.
+
+The marker gene list was created using gene lists from two publications:
+
+1. https://doi.org/10.1158/2767-9764.CRC-23-0027 - Marker genes were obtained from the list in [Supplemental Methods under `Cell annotation using external data`](https://aacr.silverchair-cdn.com/aacr/content_public/journal/cancerrescommun/3/10/10.1158_2767-9764.crc-23-0027/1/crc-23-0027-s01.pdf?Expires=1716815671&Signature=JgkH5QWs1J0dVeZFTCnZzfi2eDDQHhGsYazaF2UGI5vhPIatv7egTlW-3974FeZ~hM5Ac0gqiq0oBqeenDNWgN0PbU30N9-8n0LrsUmLHA2kCzhBxiHv8D-eKPXL0dSA9H3NKb~PkqTvOeV2QSde~MpCQL~cFh2Yn2Uf9XtAwrXrGhL3nOLIdb~qWBWtQUf1u0w~zynaDsz1hMh32gV5dEQVvZ0XZ4ozc9NO8neUbNvRJk5SSgAIPmQk1WXHKZpIheUbGU2VI2HeggkprVHO0lUELcgb1MmtziupgAkjE-xRrRUtbPVLe8QTWny6QtgJ6IcoH120LbFtRIfL9nOpuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA).
```

Comment:
Maybe put a short reference here as well? I also checked, you don't need the full url you pasted in. Most of that seems to be for tracking/login. A good tip for urls like this is to see if you can delete everything from the `?` onward. Usually you can.
```suggestion
1. Visser _et al._ (2023) https://doi.org/10.1158/2767-9764.CRC-23-0027 - Marker genes were obtained from the list in [Supplemental Methods under `Cell annotation using external data`](https://aacr.silverchair-cdn.com/aacr/content_public/journal/cancerrescommun/3/10/10.1158_2767-9764.crc-23-0027/1/crc-23-0027-s01.pdf).
```